grpcweb/src/main/scala/scalapb/grpc/grpc.scala

```diff
@@ -140,22 +141,30 @@ object ClientCalls {
         }
       )
       .on("end", { _: Any => responseObserver.onCompleted() })
-  }
 
   def asyncUnaryCall[ReqT, RespT](
       channel: Channel,
       method: MethodDescriptor[ReqT, RespT],
       options: CallOptions,
       request: ReqT
-  ): Future[RespT] = ???
+  ): Future[RespT] =
```

Comment:
Not sure if this change and the ones below are intended? Since they had no implementation I believe their purpose was just to make it compile.